gobblin-service/src/main/java/org/apache/gobblin/service/modules/scheduler/GobblinServiceJobScheduler.java

```diff
@@ -467,6 +467,16 @@ protected static String jobSchedulerTracePrefixBuilder(Properties jobProps) {
         jobProps.getProperty(ConfigurationKeys.FLOW_GROUP_KEY, "<<no flow group>>"));
   }
 
+  /**
+   * Takes a given Date object and converts the timezone to UTC before returning the number of millseconds since epoch
+   * @param date
+   */
+  public static long asUTCEpochMillis(Date date) {
```

Comment:
if `public` for more than just testability, it may belong in a utility class rather than here